src/vtl.ts

```diff
@@ -1065,7 +1071,7 @@ export abstract class VTL {
         !!array
       );
 
-      if (isBindingPattern(value.name)) {
+      if (value && isBindingPattern(value.name)) {
```

Comment:
`value?.name`

Reply:
Oddly, even though the guard asserts a type, typescript doesn't narrow the value.